### PR TITLE
update policy agent v2.2.0 docs

### PR DESCRIPTION
**Pull request by ahsayde**
Closes

**What changed?**
- update policy version v2.2.0 documents
- update policy agent profile new release

**Why was this change made?**
- to make docs up to date.

**How was this change implemented?**

**How did you validate the change?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: website/docs/policy/releases.mdx [hunk 1]
```diff
@@ -8,6 +8,32 @@ import TierLabel from "../_components/TierLabel";
...
 # Profile Releases <TierLabel tiers="enterprise" />
 
+
+## v0.6.5
+
+### Highlights
+
+- **Agent**
+  - Add support for mutating violating resource.
+  - Add end-to-end tests.
```
Comment (serboctor) [inline website/docs/policy/releases.mdx:18]: I dont think we need to mention the tests in the public docs.

**Review by serboctor — CHANGES_REQUESTED** [review]
We need to add it to the versioned docs as well so it appears on the live website.

**Review by waleedhammam — APPROVED** [review]

File: website/docs/workspaces/view-workspaces.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+<h1>
+  {frontMatter.title} <TierLabel tiers="Enterprise" />
+</h1>
+
+## Workspaces List View
+
+From the side menu. You can click on the **Workspaces** tab to go to the workspaces list view.
+
+In this view all the workspaces accross all clusters will be listed there. You can filter workspaces by its cluster or by its name.
```
Comment (serboctor) [inline website/docs/workspaces/view-workspaces.mdx:17]:
```suggestion
This view lists workspaces across all clusters. You can filter workspaces by their clusters or their names.
```

File: website/docs/workspaces/view-workspaces.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import TierLabel from "./../_components/TierLabel";
+
+<h1>
+  {frontMatter.title} <TierLabel tiers="Enterprise" />
+</h1>
+
+## Workspaces List View
+
+From the side menu. You can click on the **Workspaces** tab to go to the workspaces list view.
```
Comment (serboctor) [inline website/docs/workspaces/view-workspaces.mdx:15]:
```suggestion
From the side menu, you can click on the **Workspaces** tab to go to the workspaces list view.
```

**Review by serboctor — CHANGES_REQUESTED** [review]
LGTM...other than copying it to the versioned docs once they're ready, I've suggested a different phrasing for one of the sentences, and some punctuation-related  suggestion.

**Review by serboctor — APPROVED** [review]

**Pull request merged**